starwars/src/components/Character.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './StarWars.css'
+
+
+function Character(props) {
+    
+    return (
+        <div data-aos="zoom-in" className="character-card">
+        
+            <div className="character-info">
+                <h3 className="char-name">{props.swProp.name}</h3>
```

Comment:
good job rendering the data like this. one way I'd recommend is to make a new `paragraph` component and map out the data since there are so many `p` tags here rendering just one thing.